internal/service/encryptionatrestprivateendpoint/resource_test.go

```diff
@@ -1,29 +1,146 @@
-//nolint:gocritic
 package encryptionatrestprivateendpoint_test
 
-// TODO: if acceptance test will be run in an existing CI group of resources, the name should include the group in the prefix followed by the name of the resource e.i. TestAccStreamRSStreamInstance_basic
-// In addition, if acceptance test contains testing of both resource and data sources, the RS/DS can be omitted.
-// func TestAccEncryptionatrestprivateendpointRS_basic(t *testing.T) {
-// 	resource.ParallelTest(t, resource.TestCase{
-// 		PreCheck:                 func() { acc.PreCheckBasic(t) },
-// 		ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
-// 		//		CheckDestroy:             checkDestroyEncryptionatrestprivateendpoint,
-// 		Steps: []resource.TestStep{ // TODO: verify updates and import in case of resources
-// 			//			{
-// 			//				Config: encryptionatrestprivateendpointConfig(),
-// 			//				Check:  encryptionatrestprivateendpointAttributeChecks(),
-// 			//			},
-// 			//          {
-// 			//				Config: encryptionatrestprivateendpointConfig(),
-// 			//				Check:  encryptionatrestprivateendpointAttributeChecks(),
-// 			//			},
-// 			//			{
-// 			//				Config:            encryptionatrestprivateendpointConfig(),
-// 			//				ResourceName:      resourceName,
-// 			//				ImportStateIdFunc: checkEncryptionatrestprivateendpointImportStateIDFunc,
-// 			//				ImportState:       true,
-// 			//				ImportStateVerify: true,
-// 		},
-// 	},
-// 	)
-// }
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/common/conversion"
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/acc"
+	"go.mongodb.org/atlas-sdk/v20240805001/admin"
+)
+
+func TestAccEncryptionAtRestPrivateEndpoint_basic(t *testing.T) {
+	resource.ParallelTest(t, *basicTestCase(t))
+}
+
+func basicTestCase(tb testing.TB) *resource.TestCase {
+	tb.Helper()
+	acc.SkipTestForCI(tb) // needs Azure configuration
+	var (
+		resourceName  = "mongodbatlas_encryption_at_rest_private_endpoint.test"
```

Comment:
can resourceNames be a global variables?

Reply:
yes!